corehq/apps/app_manager/management/commands/add_form_translation_fields.py

```diff
@@ -0,0 +1,53 @@
+from corehq.apps.app_manager.management.commands.helpers import AppMigrationCommandBase
+from corehq.apps.app_manager.models import Application
+
+from dimagi.utils.chunked import chunked
+from corehq.util.log import with_progress_bar
+
+
+class ResourceNotFound(object):
+    pass
+
+
+class Command(AppMigrationCommandBase):
+    help = """
+    Populate new 'submit_label' and 'submit_notification_label' in the FormBase class with language translations
+    """
+
+    def get_apps(self):
+        return Application.objects.all()
```

Comment:
`Application` is a couch model, not Django, so you'll probably need to use something from `app_manager/dbaccessors` to get them.  I think `AppMigrationCommandBase` should be able to handle that too